src/client.ts

```diff
@@ -45,6 +48,11 @@ export interface TequilapiClient {
   connectionCancel (): Promise<void>,
   connectionIP (timeout?: number): Promise<ConnectionIPDTO>,
   connectionStatistics (): Promise<ConnectionStatisticsDTO>,
-  location (timeout?: number): Promise<ConsumerLocationDTO>,
-  sessionsList (): Promise<SessionDTO[]>
+
+  sessionsList (): Promise<SessionDTO[]>,
```

Comment:
this should probably be named sessionList, like the serviceList below it.

Reply:
Not my scope

Reply:
Not my scope